apps/dashboard/forms.py

```diff
@@ -82,6 +87,13 @@ def __init__(self, *args, team=None, **kwargs):
                 (label_to_value[label], label) for label in available_platform_labels if label in label_to_value
             ]
             self.fields["channels"].choices = platform_choices
+            all_participants = Participant.objects.filter(team=team).values("id", "identifier")
+            self.fields["participants"].choices = [(p["id"], p["identifier"]) for p in all_participants]
```

Comment:
```suggestion
            self.fields["participants"].queryset = Participant.objects.filter(team=team).select_related("user").order_by("identifier")
```

(similar for tags - you can override `__str__` on the Tag model if necessary)

Reply:
The options list works fine without overriding the __str__ method. The names are shown on the UI and the ID is sent in the filter params.